File: 준영/LV1/음양 더하기.py
def solution(absolutes, signs):
    answer = 0
    n = len(absolutes)
    for i in range(n):
        if signs[i]:
            answer += absolutes[i]
        else:
            answer -= absolutes[i]

    return answer


absolutes = [4,7,12]
signs = [True,False,True]	
print(solution(absolutes, signs))


absolutes = [1,2,3]	
signs = [False,False,True]
print(solution(absolutes, signs))

File: 태성/LV1/0604/음양더하기.js
function solution(absolutes, signs) {
    var answer = 0
    
    for (let i=0; i<signs.length; i++) {
        if(signs[i] === true) {
            signs[i] = 1
        } else {
            signs[i] = -1
        }
    }
    
    for (let i=0; i<signs.length; i++) {
        answer += absolutes[i]*signs[i]
    }
    return answer
}